gitpod-io/gitpod
# [dashboard] ConfigCat experiment

## Description
Sets up ConfigCat for the dashboard in two environments: prod, staging, and preview environments.

## Related Issue(s)

Fixes #

## How to test
1. Join this `gitpod` team https://laushinka-bbabd8f6c4.preview.gitpod-dev.com/teams/join?inviteId=148246c6-8a08-4a0d-9b51-91e59a918e73.
2. Go to /workspaces, open the console.
3. You should be able to see "Is experiment enabled?  true"

## Release Notes

```release-note
```

## Documentation
